scipy/stats/_fit.py

```diff
@@ -1216,6 +1218,42 @@ def _kolmogorov_smirnov(dist, data):
     return np.maximum(Dplus, Dminus)
 
 
+def _corr(X, M):
```

Comment:
I think I can replace this with NumPy `corrcoef`. Before, I thought it would insist on computing all pairwise coefficients but maybe not.